source/highavailability/dhcp-failover-troubleshooting.rst

```diff
@@ -15,6 +15,8 @@ Troubleshooting DHCP Failover
    and secondary nodes match.
 #. Stop and restart the DHCP daemon from **Status > Services** on both
    nodes and check the status after a few moments
+#. Check the nodes VIP status. The master node should have a value below
```

Comment:
Rather than `Check the nodes VIP status.` it should avoid making that possessive, and since the status doesn't matter so much as the configuration, perhaps this:  `Check the CARP VIP configuration for DHCP server interfaces`.

Also change `should` on this line and the next to `must`.